packages/create-plugin/README.md

```diff
@@ -20,7 +20,7 @@ Create Grafana plugins with ease.
 
 - [Plugin Tools docs](https://grafana.github.io/plugin-tools/)
 - [Plugin developer docs](https://grafana.com/docs/grafana/latest/developers/plugins/)
-- [Plugin migration guide](https://grafana.com/docs/grafana/latest/developers/plugins/migration-guide/)
+- [Plugin migration guide](../../docusaurus/docs/migration-guides/)
```

Comment:
This is the README markdown file that is part of `@grafana/create-plugin`. It ships with the package and is used by the [npm registry](https://www.npmjs.com/package/@grafana/create-plugin). These links cannot be relative to the github repo they should point to a full website url.

Reply:
Fixed.